packages/components/data-table/README.md

```diff
@@ -70,10 +70,9 @@ return <DataTable rows={rows} columns={columns} />;
 | `renderItem`           | `func`    | -        | -       | Callback function to render the content of cells under this column, overriding the default `itemRenderer` prop of the table. The `row` object is passed as an argument to this fn.                                                                                                                                                                                                                                                                                    |
 | `isTruncated`          | `boolean` | -        | `false` | Set this to `true` to allow text content of this cell to be truncated with an ellipsis, instead of breaking into multiple lines. Read the note about this option below.                                                                                                                                                                                                                                                                                               |
 | `isSortable`           | `boolean` | -        | `false` | If this is `true`, this column's header will be a become a button which calls `onSortChange` when clicked. You should enable this flag for every column you want to be able to sort. When at least one column is sortable, the table props `sortBy`, `sortDirection` and `onSortChange` should be set.                                                                                                                                                                |
-| `shouldIgnoreRowClick` | `boolean` | -        | `false` | Set this to `true` to prevent click event propagation for this cell. You might want this if you need the column to have its own `onClick` while the row also has a defined `onRowClick`.                                                                                                                                                                                                                                                                              |
+| `shouldIgnoreRowClick` | `boolean` | -        | `false` | Set this to `true` to prevent click event propagation for this cell. You might want this if you need the column to have its own call-to-action or input while the row also has a defined `onRowClick`.                                                                                                                                                                                                                                                                |
```

Comment:
How is this different to a isDisabled?

Reply:
I don't want to convey the wrong idea of disabling the cell altogether (it may contain inputs, buttons, etc, which won't be disabled).
This prop really only exists to ignore the `onRowClick` for specific columns.

Reply:
Ah. Got it.

Reply:
Thanks :)